### PR TITLE
[PT-1158] Magento2 plugin trying to confirm a declined order

### PR DESCRIPTION
**Pull request by ivan-pugach-mondu**
## Description

Minor bugfix, check whether the order was confirmed or not, before sending the "confirm" request

Release: p
Tickets: PT-1158

[Fixes # (issue)](https://mondu.atlassian.net/browse/PT-1158)

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "mondu_gmbh/magento2-payment",
   "description": "Mondu payment method for magento 2",
   "type": "magento2-module",
-  "version": "2.3.4-RC2",
+  "version": "2.3.5",
```
Comment (tikohov20) [inline composer.json:5]: lets change the version to 2.3.3

**Review by tikohov20 — COMMENTED** [review]
Looks good, lets just change the version.

File: composer.json (hunk 1 above)
Comment (ivan-pugach-mondu) [reply composer.json]: @tikohov20 updated

**Review by tikohov20 — APPROVED** [review]
utACK

**Pull request merged**